doc/source/serve/key-concepts.md

```diff
@@ -24,91 +25,100 @@ class MyFirstDeployment:
   def __call__(self, request):
       return self.msg
 
-  def other_method(self, arg):
-      return self.msg
-
 my_first_deployment = MyFirstDeployment.bind("Hello world!")
+handle = serve.run(my_first_deployment)
+print(ray.get(handle.remote())) # "Hello world!"
 ```
 
-Deployments can be exposed in two ways: exposed to an end user over HTTP, or 
-exposed to other deployments in Python by including them in the input argument of `.bind()` for other deployments,
-e.g. `deployment_1.bind(deployment_2)`.
-By default, HTTP requests will be forwarded to the `__call__` method of the class (or the function) and a `Starlette Request` object will be the sole argument.
-You can also define a deployment that wraps a FastAPI app for more flexible handling of HTTP requests. See {ref}`serve-fastapi-http` for details.
+(serve-key-concepts-query-deployment)=
+
+## ServeHandle (composing deployments)
 
-To deploy multiple deployments that serve the same class or function, use the `name` option:
+Ray Serve enables flexible model composition and scaling by allowing multiple independent deployments to call into each other.
+When binding a deployment, included in the arguments can be references to _other bound deployments_.
+Then, at runtime this argument is converted to a {mod}`ServeHandle <ray.serve.handle.RayServeHandle>` that can be used to query the deployment using a Python-native API.
```

Comment:
When binding a deployment, included in the arguments, can be ....

Reply:
thanks, will fix!